src/java/us/kbase/mobu/tester/ConfigLoader.java

```diff
@@ -47,9 +52,13 @@ public ConfigLoader(Properties props, boolean testMode,
         String tokenString = props.getProperty(testPrefix + "token");
         if (tokenString != null && tokenString.trim().isEmpty()) {
             tokenString = null;
+        }
+        if (tokenString == null) {
```

Comment:
This still seems to reverse the precedence to my eyes. Old code:
```
ts = getTokenFromConfig();
tt = getTokenFromEnv();
if tt:
  ts = tt
```
New code:
```
ts = getTokenFromConfig();
if not ts:
  tt = getTokenFromEnv();
  ts = tt
```

so in other words old code precedence env > file, new precedence file > env.

Not that that's bad, I think that's the right way to go.

Reply:
Yes, that's correct. Sorry, I misunderstood what you were saying before.